dashboard/app/views/devise/sessions/_login.html.haml

```diff
@@ -1,3 +1,7 @@
+
+- content_for :head do
```

Comment:
I know you've explained this multiple times but remind me why this needs to be in the head? (I'm getting all turned around lol)

Reply:
I've started adding any Statsig JS in the head of the document so it loads first especially when the page might be the first a user visits (and we would want to make sure Statsig has time to initialize). I was imagining this is the kind of page could be linked in SEO, so am adding to the head!